package.json

```diff
@@ -232,7 +232,7 @@
     "text-loader": "^0.0.1",
     "url-search-params": "^0.10.0",
     "us-forms-system": "https://github.com/usds/us-forms-system.git#84887dc7cb0ccee2ae2c8156b5664697c2850b1d",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#364656a5065e44f695cd631deaeb6d087674d1d7",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#46f758005675cfa61db71d8e720bdbf8387eec1e",
```

Comment:
is the corresponding yarn.lock update optional?

Reply:
Thanks for the reminder @erikphansen , updated.